swap_meet/vendor.py

```diff
@@ -0,0 +1,96 @@
+# from item import Item 
+
+class Vendor:
+
+    def __init__(self, inventory = []):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        else:
+            return False
+
+    def get_by_category(self, category):
+        self.category = category
+        category_list = []
+
+        for item in self.inventory:
+
+            if item.category == category:
+                category_list.append(item)
+
+        return category_list
+
+    def swap_items(self, other, my_item, their_item):
+        
+        self.other = other
+        self.my_item = my_item
+        self.their_item = their_item
+
+        if my_item == True and their_item == True:
+            self.inventory.remove(my_item)
+            self.inventory.append(their_item)
+            other.inventory.append(my_item)
+            other.inventory.remove(their_item)
+            return self.inventory, bool(self.inventory)
+
+        else:
+            return False
+
+    def swap_first_item(self, other):
+
+        self.other = other
+
+        if len(self.inventory) > 0 and len(other.inventory) > 0:
+            my_first_item = self.inventory[0]
+            their_first_item = other.inventory[0]
+
+            self.inventory[0] = their_first_item
+            other.inventory[0] = my_first_item
+            return bool(self.inventory)
```

Comment:
I'm curious to know why this is written this way. This will return true as long as the list isn't empty, and there's in if check above to already ensure the list isn't empty, so this should always return True anyway.
```suggestion
            return True
```